masterlist.yaml

```diff
@@ -576,8 +587,14 @@ plugins:
     clean:
       - crc: 0xB8A1EE4E
         util: SSEEdit v3.2.1
-  - name: 'static mesh improvement mod se.esp'
```

Comment:
Was this wrong, or did the plugin name just change?

Reply:
I don't know... I'd err on the side of adding another entry for SMIM-SE.esp to support both.

Reply:
Okay will restore that one and and a new entry.